main.tf

```diff
@@ -97,3 +97,99 @@ module "aws_deploy-uat-eu-north-1" {
 
   depends_on = ["${module.aws_deploy-us-west-2.static_node_ips}"]
 }
+
+module "aws_deploy-uat_mon-ap-southeast-1" {
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v1.1.1"
+  env               = "uat_mon"
+  color             = "blue"
+  bootstrap_version = "${var.bootstrap_version}"
+  vault_role        = "ae-node"
+  vault_addr        = "${var.vault_addr}"
+
+  static_nodes = 0
+  spot_nodes   = 1
+
+  spot_price    = "0.125"
+  instance_type = "m4.large"
```

Comment:
This is non-mining environment, so consider something cheaper, like t3?

Reply:
I lowered the bid to 0.03 for a t3.medium (which seems available on all regions for a price <0.02) and relatively less chance to reclaim